app/assets/stylesheets/shareable.scss

```diff
@@ -0,0 +1,10 @@
+nav.share {display:inline-block;} /* shareable's nav tag */
```

Comment:
Use `//` comments everywhere<br>Avoid qualifying class selectors with an element.<br>Colon after property should be followed by one space